storefront/app/views/layouts/workarea/storefront/application.html.haml

```diff
@@ -102,7 +102,7 @@
             .breadcrumbs{ role: 'navigation' }
               = yield :breadcrumbs
           - elsif @breadcrumbs.present?
-            = render_schema_org(breadcrumb_list_schema(@breadcrumbs.map { |t| [t.name, storefront_url_for(t)] }))
+            = render_schema_org(breadcrumb_list_schema(@breadcrumbs.map { |t| [t.name, breadcrumb_url_for(t)] }))
```

Comment:
It feels like the problem should be handled more gracefully upstream so we don't have to do all the string munging to figure out the route. Where is the bad data coming from?